README.md

```diff
@@ -164,7 +164,7 @@ package with features:
   ```yaml
   expirationd:
     task_name1:
-      space_id: 579
+      space: 579
```

Comment:
So for now, it will be `space_id` options field and corresponding `space` role config field?

Reply:
Just `space` as param-name for `expirationd.start()` and `space` as a task param for the role.

I think we can rename until we don't do a release with the role.

Reply:
See https://github.com/tarantool/expirationd/pull/129#discussion_r942384194